### PR TITLE
Update readme with development setup instructions

### PR DESCRIPTION
**Pull request by mbtamuli**
Fix for issue #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+   working on the project
+```
+source ~/.virtualenvs/pycon/bin/activate
+```
+
+4. To start working you need to check out the development branch,
+```
+git clone https://github.com/PyConPune/pune.pycon.org.git
+cd pune.pycon.org
+git checkout feature/django
```
Comment (sayanchowdhury) [inline README.md]: You need to pull the branch before checkout

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+virtualenv --python=`which python3` ~/.virtualenvs/pycon
+```
+
+3. Then activate this virtualenv, you have to do this everytime you start
+   working on the project
+```
+source ~/.virtualenvs/pycon/bin/activate
+```
+
+4. To start working you need to check out the development branch,
```
Comment (sayanchowdhury) [inline README.md]: Remove the trailing comma

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Comment by sayanchowdhury** [comment]
The PR should be to merge into 2018 branch and not feature/django branch.

**Comment by sayanchowdhury** [comment]
@mbtamuli You need to write the code on top of 2018 branch, right now you checked out your code out of feature/django branch

**Comment by mbtamuli** [comment]
Okay. Will update.

File: README.md (hunk 1 above)
Comment (mbtamuli) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
...
+   working on the project
+```
+source ~/.virtualenvs/pycon/bin/activate
+```
+
+4. To start working you need to check out the development branch
+```
+git clone https://github.com/PyConPune/pune.pycon.org.git
+cd pune.pycon.org
+git pull
```
Comment (sayanchowdhury) [inline README.md]: Should it be `git pull` or `git fetch`?

Comment (mbtamuli) [reply README.md]: Okay. Making it `git fetch`.

**Pull request merged**